wantedly/nginx-image-server
# Rename and modify script/*

## WHAT
### New Features

* Add build script `script/build`

### Changes

* Rename `script/run-target` to `script/run`
* Rename `script/run-tester` to `script/run-infrataster`
* Use bash instead of sh
* Use `latest` tag for image to ensure to use latest image
* Update `script/test` and `wercker.yml` to use renamed script

## WHY
To observe this convention:

* main container's run script should be named as `script/run`
* support container's run script should be named as `script/run-<container-name>`
* container name should be same as image name
 * e.g. quay.io/wantedly/nginx-image-server => container name: `nginx-image-server`
